code-samples/creating-your-bot/configuration-files/index.js

```diff
@@ -7,7 +7,9 @@ client.once('ready', () => {
 });
 
 client.on('message', message => {
-	if (message.content === '!ping') {
+	if (!message.content.startsWith(config.prefix)) return;
+	
```

Comment:
Please remove the trailing space and for the future I'd suggest running the lint script to prevent things like these. `yarn run lint` | `npm run lint`
```suggestion

```

Reply:
Thanks @almostSouji, I had been doing this entirely from the web-ui.